doc/developer/parallel_coding.rst

```diff
@@ -27,23 +27,28 @@ are expected to do next.
 
 Here is an example::
 
+    from armi import context
+
+    # cmd = "something different in each process"
```

Comment:
Instead of this bit of pseudocode, can we just add some actual code here that sets the values on the different processors? That way the script would be run-able and actually produce the values in the table below?

Reply:
Sure, I can do that.